ConfigSpace/forbidden.py

```diff
@@ -210,8 +214,7 @@ def is_forbidden(self, instantiated_hyperparameters, strict=True):
         # outcomes
         evaluations = []
         for component in self.components:
-            e = component.is_forbidden(instantiated_hyperparameters,
-                                       strict=strict)
+            e = component.is_forbidden(instantiated_hyperparameters)
```

Comment:
Yes, but strict is a boolean variable that is passed to `is_forbidden` in line 195; so it won't change till down here.